tests/kafkatest/tests/streams/streams_smoke_test.py

```diff
@@ -49,8 +49,8 @@ def __init__(self, test_context):
     @cluster(num_nodes=8)
     @matrix(processing_guarantee=['exactly_once_v2', 'at_least_once'],
             crash=[True, False],
-            metadata_quorum=quorum.all_non_upgrade)
-    def test_streams(self, processing_guarantee, crash, metadata_quorum=quorum.zk):
+            metadata_quorum=[quorum.combined_kraft])
```

Comment:
Can we remove `metadata_quorum` as parameter entirly ?

Reply:
Required to indicate to use KRaft for metadata quorum.  Failing to do so without a `@matrix` annotation results in this error as the test will default to ZK without it.
```File "/opt/kafka-dev/tests/kafkatest/services/kafka/kafka.py", line 541, in security_config
    zk_sasl=self.zk.zk_sasl if self.quorum_info.using_zk else False, zk_tls=self.zk_client_secure,
AttributeError: 'NoneType' object has no attribute 'zk_sasl'
```